### DISCUSSION TITLE
Notifying about the meeting schedule

### DISCUSSION THREAD

**Discussion by vyasr** [scikit-build/scikit-build#898] [General]
Are the planned dates and times for the scikit-build dev meetings currently posted anywhere? I sometimes see a notification on this discussion board that a meeting is upcoming, but those notifications are typically only a day in advance of the meeting and I have yet to notice them in time. As I am not a member of the org I don't believe that I can subscribe to notifications on these in any useful way (I do have my watcher settings set up for the individual repos, but I don't see any information about how to configure org discussion notifications on the [GH docs for notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) for organizations of which I am not a member) but please let me know if I am wrong. Thank you!

**Comment by henryiii** [comment]
We should be better about posting the schedule - it's the third Friday of every month, noon Eastern. The next one is April 21. And May 19 after that. Probably should update the main pages of scikit-build and scikit-build-core, or start working on scikit-build.org soon.

**Reply by vyasr** [reply]
Got it, thanks for the heads up. I'll remember that one for next month.
